File: frontEnd/src/types/dto/stepDTO/HoverStepDTO.ts
// 悬停操作
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {ElementDTO} from "@/types/dto/ElementDTO";

export class HoverStepDTO extends StepBaseDTO {
    stepType: string = "HOVER"; // 步骤类型
    element: ElementDTO = new ElementDTO(); // 元素
}

File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/HoverStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// 点击操作

import com.mokatest.platform.demos.domain.ui.dto.ElementDTO;
import lombok.Data;

@Data
public class HoverStepDTO extends StepBaseDTO {
    private String stepType; // 步骤类型
    private ElementDTO element; // 元素
}